protobuf/plugin/raftproxy/raftproxy.go

```diff
@@ -49,21 +49,27 @@ func (g *raftProxyGen) genProxyConstructor(s *descriptor.ServiceDescriptorProto)
 		md["redirect"] = append(md["redirect"], addr)
 		return metadata.NewContext(ctx, md), nil
 	}
-	mods := []func(context.Context)(context.Context, error){redirectChecker}
-	mods = append(mods, ctxMod)
+	remoteMods := []func(context.Context)(context.Context, error){redirectChecker}
```

Comment:
sounds good as it is. I mostly meant `append` vs creating new slice. I think it's okay, though.